cmd/podman/logs.go

```diff
@@ -29,6 +26,15 @@ var (
 			logsCommand.GlobalFlags = MainGlobalOpts
 			return logsCmd(&logsCommand)
 		},
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
Nice! Since this now takes multiple args, could you change `Use` to `CONTAINER [CONTAINER...]` ? Thanks.